app/src/main/java/com/mk/tasky/core/presentation/TaskyBackground.kt

```diff
@@ -18,31 +18,46 @@ import com.mk.tasky.ui.theme.White
 
 @Composable
 fun TaskyBackground(
-    @StringRes title: Int,
+    @StringRes title: Int? = null,
```

Comment:
Naming recommendation: `titleResId`, else it'll sound like a string

Reply:
Awesome. Fixed!